### DISCUSSION TITLE
Excited for ToonSquid 1.8

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#140] [General]
ToonSquid has 8 updates about 1.7 so far. I’m so hype and curious to know what’s coming next! I know Keiwan has lots and lots of Ideas to implement in the app. What do you think it’s coming earlier? :DD

**Comment by dabing1022** [comment]
- layer depth: I wish the layer depth, so when the camera moves, the background/forground layers moves at different speeds!

**Reply by DinoAwakening** [reply]
So you mean you want Parallax to be added in ToonSquid? Isn’t that like a 3D thing? xd

**Reply by dabing1022** [reply]
Like AnimateCC maybe 

https://helpx.adobe.com/animate/using/layer-depth.html

**Reply by fuki98** [reply]
i want this :) +1

**Reply by DWrightAnimation** [reply]
I wish there was a tutorial for how to do this correctly without the need of a 3D landscape.

**Comment by DinoAwakening** [comment]
Keira do didn’t disappoint us with 1.8 for sure. But is to be known what is coming next on 1.9 :DD
